pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -43,20 +43,28 @@ var ManifestPath = "./bindata"
 // Add creates a new OperConfig Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager, status *statusmanager.StatusManager) error {
-	return add(mgr, newReconciler(mgr, status))
+	reconciler, err := newReconciler(mgr, status)
+	if err != nil {
+		return err
+	}
+	return add(mgr, reconciler)
 }
 
 // newReconciler returns a new reconcile.Reconciler
-func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager) *ReconcileOperConfig {
-	configv1.Install(mgr.GetScheme())
-	operv1.Install(mgr.GetScheme())
+func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager) (*ReconcileOperConfig, error) {
+	if err := configv1.Install(mgr.GetScheme()); err != nil {
+		return nil, err
+	}
+	if err := operv1.Install(mgr.GetScheme()); err != nil {
+		return nil, err
+	}
```

Comment:
`configv1.Install()` and `operv1.Install()` return errors to conform to a particular type signature, but they can't actually fail. You don't need to rewrite all this stuff. Just ignore the errors. (Likewise in the next commit.)

Reply:
fixed in updated patchset